package.json

```diff
@@ -18,7 +18,7 @@
     "dts-check": "tsc --project tests/types/tsconfig.json",
     "lint": "standard",
     "pretest": "npm run lint && npm run dts-check",
-    "test": "lab tests/* --coverage"
```

Comment:
Caused `Could not find test file or directory 'tests/*'.` problem when running `npm test`